src/project/FileTreeView.js

```diff
@@ -181,9 +182,9 @@ define(function (require, exports, module) {
             var fullname = this.props.name,
                 extension = LanguageManager.getCompoundFileExtension(fullname);
 
-            var node = this.refs.name.getDOMNode();
+            var node = this.refs.name;
             node.setSelectionRange(0, _getName(fullname, extension).length);
-            ViewUtils.scrollElementIntoView($("#project-files-container"), $(node), true);
+            ViewUtils.scrollElementIntoView($("#project-files-container"), $(ReactDOM.findDOMNode(node)), true);
```

Comment:
Good catch: this was a brainfart due to it needing being needed @ L365 on `componentDidUpdate` because it targets `this`. Fixed.